src/components/map/test/unit/vl-map-wmts-layer.test.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="/node_modules/web-component-tester/browser.js"></script>
+
+    <script type="module" src="../../../../../lib/index.js"></script>
+
+    <body>
+      <test-fixture id="vl-map-wmts-layer-fixture">
+        <template>
+          <vl-map>
+            <vl-map-wmts-layer
+              data-vl-url="https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts"
+              data-vl-layer="grb_sel"
+              data-vl-name="GRB Wegenkaart"
+              data-vl-min-resolution="2"
+              data-vl-max-resolution="4"
+            >
+            </vl-map-wmts-layer>
+          </vl-map>
+        </template>
+      </test-fixture>
+
+      <test-fixture id="vl-map-wmts-layer-hidden-fixture">
+        <template>
+          <vl-map>
+            <vl-map-wmts-layer
+              data-vl-url="https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts"
+              data-vl-layer="grb_sel"
+              data-vl-name="GRB Wegenkaart"
+              data-vl-min-resolution="2"
+              data-vl-max-resolution="4"
+              data-vl-hidden
+            >
+            </vl-map-wmts-layer>
+          </vl-map>
+        </template>
+      </test-fixture>
+
+      <test-fixture id="vl-map-fixture">
+        <template>
+          <vl-map></vl-map>
+        </template>
+      </test-fixture>
+
+      <script type="module">
+import {
+  OlWMTSSource,
+  OlWMTSTileGrid,
+} from "../../../../../lib/components/map/mapactions/index.js";
+suite("vl-map-wmts-layer", () => {
+  const LAYER_SELECTOR = "[data-vl-is-layer]";
+
+  const getLayer = (map) => {
+    return map.querySelector(LAYER_SELECTOR);
+  };
+
+  setup((done) => {
+    customElements.whenDefined("vl-map-wmts-layer").then(() => done());
+  });
+
+  test("de wmts source wordt correct geconfigureerd", async () => {
+    const map = fixture("vl-map-wmts-layer-fixture");
+    await map.ready;
+
+    const layers = map.map.getOverlayLayers();
+    assert.lengthOf(layers, 1);
+    const layer = layers[0];
+
+    const source = layer.getSource();
+    assert.isTrue(source instanceof OlWMTSSource);
+
+    assert.lengthOf(source.urls, 1);
+    assert.equal(
+      source.urls[0],
+      "https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts"
+    );
+    assert.equal(source.getLayer(), "grb_sel");
+    assert.equal(source.getMatrixSet(), "BPL72VL");
+    assert.equal(source.getFormat(), "image/png");
+    assert.equal(
+      JSON.stringify(source.getProjection()),
+      JSON.stringify(map._projection)
+    );
+    assert.equal(source.getStyle(), "");
+
+
+    // TODO moet dit getest worden? Is volledig afhankelijk van server.
```

Comment:
Deze TODO nog fixen. Kan met behulp van fecthMock, maar ik heb dat niet aan de praat gekregen. Indien er een voorbeeld XML nodig is, kan ik die wel aanleveren.